cmd/goQuery/commands/root.go

```diff
@@ -118,10 +117,8 @@ func entrypoint(cmd *cobra.Command, args []string) error {
 	// run commands that don't require any argument
 	// handle list flag
 	if cmdLineParams.List {
-		err := listInterfaces(cmdLineParams.DBPath, cmdLineParams.External)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to retrieve list of available databases: %s", err)
-			return err
+		if err := listInterfaces(cmdLineParams.DBPath, cmdLineParams.External); err != nil {
```

Comment:
Can we agree to leave each other alone when it comes to the `err; if err` controversy? Unless the OCD trigger is above 9000 an we need to settle the debate and do it consistently across the code base.

Reply:
Would also appreciate it if it isn't touched until https://github.com/els0r/goProbe/issues/43 is done